binders/kafka-binder/spring-cloud-stream-binder-kafka-reactive/src/test/java/org/springframework/cloud/stream/binder/reactorkafka/ReactorKafkaBinderIntegrationTests.java

```diff
@@ -199,7 +199,7 @@ Function<Flux<ReceiverRecord<byte[], byte[]>>, Flux<String>> lowercase() {
 		@Bean
 		java.util.function.Consumer<Flux<String>> patternConsumer() {
 			return f -> f.doOnNext(s -> patternedDeliveries.add(s))
-					.subscribe();
+				.subscribe();
```

Comment:
Does not look like the change in this class is meaningful.
Might be just fully reverted since PR is already complex enough.